modules/accredited_representative_portal/app/models/accredited_representative_portal/representative_user_account.rb

```diff
@@ -18,6 +18,26 @@ def active_power_of_attorney_holders
         .select(&:accepts_digital_power_of_attorney_requests?)
     end
 
+    def power_of_attorney_holders
+      @power_of_attorney_holders ||= registrations.flat_map do |registration|
+        number = registration.accredited_individual_registration_number
+        type = registration.power_of_attorney_holder_type
+
+        case type
+        when PowerOfAttorneyHolder::Types::VETERAN_SERVICE_ORGANIZATION
+          ##
+          # Other types are 1:1 and will have no reason to introduce a
+          # complicated method that takes a block like this.
+          #
+          get_organizations(number) do |attrs|
+            PowerOfAttorneyHolder.new(type:, **attrs)
+          end
+        else
+          []
+        end
+      end
+    end
+
```

Comment:
@patrick-brown-oddball Make sure to check your diffs in the future. I don't think there is a reason for this pure within-file move. Any change can make it slightly more burdensome to do investigations that require git history to see why some code is the way it is.